### DISCUSSION TITLE
Link to 0.27.1 apk please?

### DISCUSSION THREAD

**Discussion by raven9nine** [TeamNewPipe/NewPipe#11286] [Questions] [answered]
### Checklist

- [X] I made sure that there are *no existing issues or discussions* - [open](https://github.com/TeamNewPipe/NewPipe/issues) or [closed](https://github.com/TeamNewPipe/NewPipe/issues?q=is%3Aissue+is%3Aclosed) - which I could contribute my information to.
- [X] I have read the [FAQ](https://newpipe.net/FAQ/) and my question isn't listed.
- [X] I have taken the time to fill in all the required details. I understand that the question will be dismissed otherwise.
- [X] I have read and understood the [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/dev/.github/CONTRIBUTING.md).

### What is/are your question(s)?

Please post a link to the version 0.27.1 apk it is not on f-droid.

**Comment by wb9688** [comment, marked as answer]
It is on our F-Droid repo and soon on upstream F-Droid as well, plus you can download it from the releases page on GitHub as always. https://github.com/TeamNewPipe/NewPipe/releases

**Reply by raven9nine** [reply]
Thankyou, much appreciated
